SimG4CMS/Calo/interface/HCalSD.h

```diff
@@ -14,6 +14,7 @@
 #include "SimG4CMS/Calo/interface/HFShowerPMT.h"
 #include "SimG4CMS/Calo/interface/HFShowerFibreBundle.h"
 #include "SimG4CMS/Calo/interface/HcalNumberingScheme.h"
+#include "SimG4CMS/Calo/interface/HcalTestNS.h"
```

Comment:
Here, the include is needed only in the c++ file rather than the header (just forward declare the class)